.mise/tasks/infrastructure/down

```diff
@@ -1,10 +1,6 @@
 #!/usr/bin/env sh
 
-# mise description="destroy infrastructure"
+#!/usr/bin/env sh
```

Comment:
Is this an extra shebang? If so, remove.

Reply:
fixed